app/models/restaurant.rb

```diff
@@ -27,4 +31,15 @@ def self.top_restaurants
   def editable_by?(current_user)
     current_user.role == "admin" || current_user == user
   end
+
+  def self.search_results(query)
+    sql = %Q{
```

Comment:
Use `%` instead of `%Q`.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.